src/main/java/net/sf/jabref/gui/specialfields/SpecialFieldIcon.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.jabref.gui.specialfields;
+
+import javax.swing.Icon;
+import javax.swing.JLabel;
+
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.specialfields.SpecialField;
+import net.sf.jabref.model.entry.SpecialFieldValue;
+
+
+public class SpecialFieldIcon {
+
+    public static Icon getRepresentingIcon(SpecialField field) {
+        if (SpecialField.PRINTED.equals(field)) {
```

Comment:
@lenhard  Yeah. Just add a fieldfrom the type Icon in the Special Field enum
then you could directly assign the Icon and we don't need a Hashmap or the equals stuff.

e.g `MY_SPEC_FIELD("asdf"...,....,IconTheme.JabRefIcon.READ_STATUS.getSmallIcon())`